packages/fxa-settings/src/components/Settings/FlowRecoveryKeyConfirmPwd/index.stories.tsx

```diff
@@ -60,7 +60,7 @@ const accountWithThrottledErrorOnSubmit = {
 } as unknown as Account;
 
 const StoryWithContext = (account: Account) => {
-  const [formattedRecoveryKey, setFormattedRecoveryKey] = useState<string>('');
+  const [, setFormattedRecoveryKey] = useState<string>('');
```

Comment:
Another array with a leading comma.

Reply:
Oh I see.  I thought there's a convention of using an underscore for unused/ignored vars.  Does that not work here?

Reply:
Let me try that. I think maybe it still complained.

Reply:
It still complains. Maybe that syntax only works with function args, or our linter rules could be adjusted somehow? Not sure... Anyways, the leading coma is actually valid.

Reply:
Alternatively,
```javascript
const setFormattedRecoveryKey = useState<string>('')[1];
```